### PR TITLE
Activate Bors!

### PR DESCRIPTION
**Pull request by Lokathor**

### CODE REVIEW

**Comment by Lokathor** [comment]
bors ping

**Comment by Lokathor** [comment]
@Dylan-DPC when you get around to this I just want you to test that you can also `bors r+` it since you're the current backup PR merger.

**Comment by Dylan-DPC** [comment]
bors: try

File: .travis.yml [hunk 1]
```diff
@@ -11,6 +11,13 @@ rust:
 cache:
   cargo
 
+branches:
+  only:
+    - staging
+    - trying
+    - master
+    - lokathor
```
Comment (Dylan-DPC) [inline .travis.yml]: you don't need to specify  custom branches here. since you are always merging to master and not to lokathor

Comment (Lokathor) [reply .travis.yml]: The guide said to use that to prevent travis and appveyor from trying to run the temp branches that bors generates while it's working

Comment (Dylan-DPC) [reply .travis.yml]: yeh those are fine, but lokathor isn't generated right? so you don't have to include that.

Comment (Lokathor) [reply .travis.yml]: Oh! I see what you mean.

But i want appveyor to tell me about a failed build I pushed :P

**Comment by Lokathor** [comment]
?bors r+

**Pull request merged**